apps/backend/src/flat-files/tdi17/TDI17Details.ts

```diff
@@ -123,7 +123,7 @@ export class TDI17Details
     this.resource.location_desc = data;
   }
 
-  @Column({ start: 67, width: 12 })
+  @Column({ start: 67, width: 12, format: { type: DataType.Decimal } })
```

Comment:
**note: I am not using float here as this requires further parsing
The string is passed in as something like: "00009900"
The last two digits are always the two decimal places, as described in the TDI definition files. We need to insert a decimal prior to parsing it as a float.